src/block-components/image/editor.scss

```
@@ -85,11 +85,21 @@
 }
 
 // Hide the size tooltip when the image resizer is too narrow.
-.stk-img-resizer {
+.stk-img-resizer-tooltip-container {
+	position: absolute;
+	inset: 0;
+	pointer-events: none;
+	// Keep the tooltip above the click-to-edit overlay at z-index 2.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)